src/modules/blocks/create-article-metadata.js

```diff
@@ -0,0 +1,60 @@
+import { toBlock, replaceElement, newHtmlList } from '../utils/dom-utils.js';
+
+export default function createArticleMetaData(
+  document,
+  meta,
+  lastUpdated,
+  level,
+) {
+  // Article Metadata Title
+  const headerElements = document.querySelector('h1');
+  const articleMetaDivTag = document.createElement('div');
+  const headerElementTag = document.createElement('h1');
+  headerElementTag.textContent = headerElements.innerHTML;
+  articleMetaDivTag.append(headerElementTag);
+
+  // Article Metadata Last Updated
+  if (lastUpdated.length !== 0) {
+    const lastUpdatedElementTag = document.createElement('div');
+    const isodate = new Date(lastUpdated);
+    const options = { year: 'numeric', month: 'long', day: 'numeric' };
+    const formattedDate = isodate.toLocaleDateString('en-US', options);
+    const localeDateString = `Last update: ${formattedDate}`;
+    lastUpdatedElementTag.innerHTML = localeDateString;
+    articleMetaDivTag.append(lastUpdatedElementTag);
+  }
+
+  // Article Metadata Topics
+  const fullMetadata = meta.split('\n');
```

Comment:
instead of this, use this package: https://www.npmjs.com/package/yaml
to parse metadata as YAML, then get the `feature` value and do your transformations on it

Reply:
Updated the code to parse Metadata as YAML and validated the changes @ahmed-musallam  @Sivaramvt